README.md

```diff
@@ -21,17 +21,21 @@ Here are some example `EdgeX Events` with accompanying `EdgeX Readings`.
 - **`InventoryEventArrived`**
 ```json
 {
+  "apiVersion": "v2",
   "id": "6def8859-5a12-4c83-b68c-256303146682",
-  "device": "rfid-llrp-inventory",
-  "created": 1598043284110,
+  "deviceName": "rfid-llrp-inventory",
+  "profileName": "rfid-llrp-inventory",
+  "sourceName" : "rfid-llrp-inventory",
   "origin": 1598043284109799400,
   "readings": [
     {
+      "apiVersion": "v2",
       "id": "8d15d035-402f-4abc-85fc-a7ed7213122a",
-      "created": 1598043284110,
       "origin": 1598043284109799400,
-      "device": "rfid-llrp-inventory",
-      "name": "InventoryEventArrived",
+      "deviceName": "rfid-llrp-inventory",
+      "resourceName": "InventoryEventArrived",
+      "profileName": "rfid-llrp-inventory",
+      "ValueType": "String",
```

Comment:
Why is `ValueType` using `PascalCase` vs the others that are using `camelCase`?

Reply:
I have made the corrections to the readme and will have the pushed shortly